### PR TITLE
Translate #15, ready for proofreading.

### PR DESCRIPTION
**Pull request by lruzicka**
Please, check the button, tab and settings descriptions as I do not have any Fegora installed and could not see the real life English translations. Thanks.

### CODE REVIEW

File: docs/handout_en.md [hunk 1]
```diff
@@ -162,33 +162,31 @@ Na ploše se též nachází soubor handout,html, který nejspíš právě čtet
...
-Zajímavým nastavením pro vás nejspíš bude výběr rozložení klávesnice.
-Tento přepínač se nachází na kartě "Obecné" a dostanete se na něj
-stisknutím TAB hned po otevření nastavení. Pokud změníte rozložení na
-Stolní počítač", klávesa Orca bude mapována na klávesu Insert a dojde i
-k jiným změnám tak, aby se Orca příjemně používala na plnohodnotné
-klávesnici s numerickým blokem. Někteří by si možná rádi změnili
-hlasovou syntézu. Toho docílíte na kartě"Hlas". Měňte pouze seznamy
-označené"Hlasový výstup\" a "osoba", v seznamu "Řečový systém" nechte
-původní hodnotu.
+## Quick introduction to the Orca reader
```
Comment (vojtapolasek) [inline docs/handout_en.md:165]:
```suggestion
## Quick introduction to the Orca screenreader
```

File: docs/handout_en.md [hunk 2]
```diff
@@ -162,33 +162,31 @@ Na ploše se též nachází soubor handout,html, který nejspíš právě čtet
...
+
+Orca has most of the functions that you might expect from a modern
+screen reader. Similary to NVDA, for example, Orca has its **special key**
+that we will refer to as the **Orca**. By default, it corresponds with
+the **CapsLock** key.
+
+As a matter of course, Orca offers a solid number of settings. The **Preferences**
+dialogue can be activated using **Orca+Space**. The cursor can be found on
```
Comment (vojtapolasek) [inline docs/handout_en.md:173]:
```suggestion
dialogue can be activated using **Orca+Space**. The focus will be placed on
```

File: docs/handout_en.md [hunk 3]
```diff
@@ -162,33 +162,31 @@ Na ploše se též nachází soubor handout,html, který nejspíš právě čtet
...
+a row of tabs which can be switched using the **right and left arrows**.
```
Comment (vojtapolasek) [inline docs/handout_en.md:174]:
```suggestion
a row of tabs which can be browsed using the **right and left arrows**. 
```

File: docs/handout_en.md [hunk 4]
```diff
@@ -162,33 +162,31 @@ Na ploše se též nachází soubor handout,html, který nejspíš právě čtet
...
+
+Orca has most of the functions that you might expect from a modern
+screen reader. Similary to NVDA, for example, Orca has its **special key**
+that we will refer to as the **Orca**. By default, it corresponds with
+the **CapsLock** key.
+
+As a matter of course, Orca offers a solid number of settings. The **Preferences**
+dialogue can be activated using **Orca+Space**. The cursor can be found on
+a row of tabs which can be switched using the **right and left arrows**. 
+The **Tab** key iterates over the control items (**Shift+Tab** in the reversed
```
Comment (vojtapolasek) [inline docs/handout_en.md:175]:
```suggestion
The **Tab** key iterates over the controls (**Shift+Tab** in the reversed 
```

File: docs/handout_en.md [hunk 5]
```diff
@@ -162,33 +162,31 @@ Na ploše se též nachází soubor handout,html, který nejspíš právě čtet
...
+Orca has most of the functions that you might expect from a modern
+screen reader. Similary to NVDA, for example, Orca has its **special key**
+that we will refer to as the **Orca**. By default, it corresponds with
+the **CapsLock** key.
+
+As a matter of course, Orca offers a solid number of settings. The **Preferences**
+dialogue can be activated using **Orca+Space**. The cursor can be found on
+a row of tabs which can be switched using the **right and left arrows**. 
+The **Tab** key iterates over the control items (**Shift+Tab** in the reversed 
+order). The buttons, check-buttons and radio-buttons can be activated using the
```
Comment (vojtapolasek) [inline docs/handout_en.md:176]:
```suggestion
order). The buttons, checkboxes and radio-buttons can be activated using the 
```

File: docs/handout_en.md [hunk 6]
```diff
@@ -162,33 +162,31 @@ Na ploše se též nachází soubor handout,html, který nejspíš právě čtet
...
+screen reader. Similary to NVDA, for example, Orca has its **special key**
+that we will refer to as the **Orca**. By default, it corresponds with
+the **CapsLock** key.
+
+As a matter of course, Orca offers a solid number of settings. The **Preferences**
+dialogue can be activated using **Orca+Space**. The cursor can be found on
+a row of tabs which can be switched using the **right and left arrows**. 
+The **Tab** key iterates over the control items (**Shift+Tab** in the reversed 
+order). The buttons, check-buttons and radio-buttons can be activated using the 
+**Space** key. Any list item is selected using the **up and down arrows**.
```
Comment (vojtapolasek) [inline docs/handout_en.md:177]:
```suggestion
**Spacebar** key. Any list item is selected using the **up and down arrows**. 
```

File: docs/handout_en.md [hunk 7]
```diff
@@ -162,33 +162,31 @@ Na ploše se též nachází soubor handout,html, který nejspíš právě čtet
...
+the **CapsLock** key.
+
+As a matter of course, Orca offers a solid number of settings. The **Preferences**
+dialogue can be activated using **Orca+Space**. The cursor can be found on
+a row of tabs which can be switched using the **right and left arrows**. 
+The **Tab** key iterates over the control items (**Shift+Tab** in the reversed 
+order). The buttons, check-buttons and radio-buttons can be activated using the 
+**Space** key. Any list item is selected using the **up and down arrows**. 
+The same way you toggle switches and move sliders. If you want to save the
+changes, mind to activate the **OK** button.
```
Comment (vojtapolasek) [inline docs/handout_en.md:179]:
```suggestion
changes, remember to activate the **OK** button.
```

File: docs/handout_en.md [hunk 8]
```diff
@@ -162,33 +162,31 @@ Na ploše se též nachází soubor handout,html, který nejspíš právě čtet
...
+
+An interesting setting for you will probably be the keyboard layout.
+This switch is located on the **General** tab and you can reach it by 
+pressing **Tab** immediately after the Settings dialogue opens. If you
+change the layout to **Desktop**, the Orca key will be moved to the **Insert**
```
Comment (vojtapolasek) [inline docs/handout_en.md:184]:
```suggestion
change the layout to **Desktop**, the Orca key will be set to the **Insert**
```

File: docs/handout_en.md [hunk 9]
```diff
@@ -162,33 +162,31 @@ Na ploše se též nachází soubor handout,html, který nejspíš právě čtet
...
+key and some other changes will be made as well to enable better 
+ergonomics on a fullsize keyboard with a numerical block. Some of you
+might want to change the voice synthesis. You can do so on the **Voice** tab.
```
Comment (vojtapolasek) [inline docs/handout_en.md:187]:
```suggestion
might want to change the voice synthesis. You can do so on the **Speech** tab.
```

File: docs/handout_en.md [hunk 10]
```diff
@@ -162,33 +162,31 @@ Na ploše se též nachází soubor handout,html, který nejspíš právě čtet
...
+Change only the **Voice input** and **Person** values and leave the **Speech
```
Comment (vojtapolasek) [inline docs/handout_en.md:188]:
```suggestion
Change only the **Speech Synthesizer** and **Person** values and leave the **Speech
```

**Review by vojtapolasek — CHANGES_REQUESTED** [review]
see my comments, accept suggestions if you think they make sense. thank you.

**Comment by lruzicka** [comment]
I have made the required changes.

**Review by vojtapolasek — APPROVED** [review]

**Comment by vojtapolasek** [comment]
Thank you for all changes. Merging.

**Pull request merged**